notification_configuration.go

```diff
@@ -93,12 +93,19 @@ type NotificationConfiguration struct {
 
 // DeliveryResponse represents a notification configuration delivery response.
 type DeliveryResponse struct {
-	Body       string      `json:"body"`
-	Code       int         `json:"code"`
-	Headers    http.Header `json:"headers"`
-	SentAt     time.Time   `json:"sent-at,iso8601"`
-	Successful bool        `json:"successful"`
-	URL        string      `json:"url"`
+	Body string `jsonapi:"attr,body"`
+	Code string `jsonapi:"attr,code"`
+	//Headers    http.Header `jsonapi:"attr,headers"`
+	SentAt     string `jsonapi:"attr,sent-at"`
+	Successful string `jsonapi:"attr,successful"`
+	URL        string `jsonapi:"attr,url"`
```

Comment:
This is waiting on a fix in the API to send this data in a proper format. Specifically relating to `SentAt` and Code`. `SentAt` is waiting on a time format update in `hashicorp/jsonapi`, and `Code` is supposed to be an integer, which will be updated in the API.